src/tests/nativeaot/SmokeTests/DwarfDump/Program.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+
+public class Program
+{
+    public static int Main(string[] args)
+    {
+        var llvmDwarfDumpPath = Path.Combine(
+            Environment.GetEnvironmentVariable("CORE_ROOT"),
+            "SuperFileCheck",
+            "runtimes",
+            RuntimeInformation.RuntimeIdentifier,
+            "native",
+            "llvm-dwarfdump");
+
+        if (!File.Exists(llvmDwarfDumpPath))
+        {
+            // Linux-only test
+            Console.WriteLine("Could not find llvm-dwarfdump at path: " + llvmDwarfDumpPath);
+            return 1;
+        }
+
+        Console.WriteLine("Running llvm-dwarfdump");
+        var proc = Process.Start(llvmDwarfDumpPath, "--version");
+
+        if (proc is null)
+        {
+            Console.WriteLine("llvm-dwarfdump could not run");
+            return 2;
+        }
+
+        proc.WaitForExit();
+        if (proc.ExitCode != 0)
+        {
+            return 3;
+        }
+
+        proc = Process.Start(new ProcessStartInfo
+        {
+            FileName = "/bin/sh",
```

Comment:
Why do we wrap this in `/bin/sh` and don't wrap the `--version` one before?

Reply:
Oops. This was a remnant from when I was using `grep` to do the filtering, instead of doing it directly in C#. I'll remove.